src/v2/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -315,6 +317,12 @@ export const ConfirmBidRoute: React.FC<
     }
   }
 
+  function renderZendeskScript() {
+    if (typeof window !== "undefined" && window.zEmbed) return
+
+    return <ZendeskWrapper zdKey={sd.AUCTION_ZENDESK_KEY} />
```

Comment:
It looks like the reason this issue happened is because `ZendeskWrapper` has no types in the file, so there was no error stating that `zdKey` was missing (and so it returned `null`. 

If the key is _actually_ null then we should def return null from the component so that things don't blow up, but we'll want to add `React.FC<{ zdKey: string }>` (a required prop) to ZendeskWrapper so that at the very least we'll get a type error if the key isn't being passed.